tests/fortinet/get_system_status/fortinet_get_system_status_7.0_eval.yml

```
@@ -0,0 +1,43 @@
+parsed_sample:
```

Start the YAML file with three dashes.
(If using the helper script, it adds these by default.)

```suggestion
---
parsed_sample:
```

---

Thanks for the suggestion!